src/System.IO.Pipes/src/System/IO/Pipes/AnonymousPipeServerStream.cs

```diff
@@ -65,9 +65,14 @@ public AnonymousPipeServerStream(PipeDirection direction, SafePipeHandle serverS
             State = PipeState.Connected;
         }
 
+        public AnonymousPipeServerStream(PipeDirection direction, HandleInheritability inheritability, int bufferSize)
+            : this(direction, inheritability, bufferSize, null)
+        {
+        }
+
         // bufferSize is used as a suggestion; specify 0 to let OS decide
         // This constructor instantiates the PipeSecurity using just the inheritability flag
-        public AnonymousPipeServerStream(PipeDirection direction, HandleInheritability inheritability, int bufferSize)
+        internal AnonymousPipeServerStream(PipeDirection direction, HandleInheritability inheritability, int bufferSize, PipeSecurity pipeSecurity)
```

Comment:
`PipeSecurity` is not available in Unix, so the build is faling in that configuration. I need to change this.

Reply:
Addressed by moving all the code consuming `PipeSecurity` inside the *.Windows.cs file. The shared file was mostly reverted (only kept the change removing the unused `using` directive).